dags/atd_service_bot_issues_to_socrata.py

```diff
@@ -0,0 +1,86 @@
+import os
+from datetime import datetime, timedelta
+
+from airflow.models import DAG
+from airflow.operators.docker_operator import DockerOperator
+
+from onepasswordconnectsdk.client import Client, new_client
+import onepasswordconnectsdk
+
+DEPLOYMENT_ENVIRONMENT = os.getenv("ENVIRONMENT", "development")
+ONEPASSWORD_CONNECT_HOST = os.getenv("OP_CONNECT")
+ONEPASSWORD_CONNECT_TOKEN = os.getenv("OP_API_TOKEN")
+VAULT_ID = os.getenv("OP_VAULT_ID")
+
+default_args = {
+    "owner": "airflow",
+    "description": "Publish atd-data-tech Github issues to Socrata",
+    "depends_on_past": False,
+    "start_date": datetime(2015, 12, 1),
+    "email_on_failure": False,
+    "email_on_retry": False,
+    "retries": 1,
+}
+
+docker_image = "atddocker/atd-service-bot:production"
+
+REQUIRED_SECRETS = {
+    "GITHUB_ACCESS_TOKEN": {
+        "opitem": "Service Bot",
+        "opfield": "shared.githubAccessToken",
+        "opvault": VAULT_ID,
+    },
+    "ZENHUB_ACCESS_TOKEN": {
+        "opitem": "Service Bot",
+        "opfield": "shared.zenhubAccessToken",
+        "opvault": VAULT_ID,
+    },
+    "SOCRATA_API_KEY_ID": {
+        "opitem": "Service Bot",
+        "opfield": "shared.socrataApiKeyID",
+        "opvault": VAULT_ID,
+    },
+    "SOCRATA_API_KEY_SECRET": {
+        "opitem": "Service Bot",
+        "opfield": "shared.socrataApiKeySecret",
+        "opvault": VAULT_ID,
+    },
+    "SOCRATA_APP_TOKEN": {
+        "opitem": "Service Bot",
+        "opfield": "shared.socrataAppToken",
+        "opvault": VAULT_ID,
+    },
+    "SOCRATA_RESOURCE_ID": {
+        "opitem": "Service Bot",
+        "opfield": f"{DEPLOYMENT_ENVIRONMENT}.socrataResourceId",
+        "opvault": VAULT_ID,
+    },
+    "SOCRATA_ENDPOINT": {
+        "opitem": "Service Bot",
+        "opfield": f"shared.socrataEndpoint",
+        "opvault": VAULT_ID,
+    },
+}
+
+client: Client = new_client(ONEPASSWORD_CONNECT_HOST, ONEPASSWORD_CONNECT_TOKEN)
+env_vars = onepasswordconnectsdk.load_dict(client, REQUIRED_SECRETS)
+
+with DAG(
+    dag_id=f"atd_service_bot_github_to_socrata_{DEPLOYMENT_ENVIRONMENT}",
+    default_args=default_args,
+    schedule_interval="21 5 * * *",
```

Comment:
Not really important, but I'm not sure I understand the reason why we pick 21 minutes after the 5th hour for this one. It seems oddly specific. If there is a reason, having a comment in here to explain that would be helpful, or otherwise, maybe `0 5 * * *` would be better. 

If the worry is that we'll have a ton of DAGs fire up at once, then we should seek to address that via the [parallelism](https://github.com/cityofaustin/atd-airflow/blob/airflow-v2/airflow.cfg#L26-L30) configuration option or one of the related configurables.

Reply:
fwiw, one of the things John told me about writing dags on airflow was to try to not schedule them to run when other dags were running, to pick different times. so if I had to guess, I'd say someone picked this arbitrary time to not step on other dag toes.